web/src/components/ClusterDistribution/ClusterDistributionPlotCard.tsx

```diff
@@ -2,28 +2,41 @@
 import React from 'react'
 
 import { Card, CardBody, CardHeader, Col, Row } from 'reactstrap'
+import styled from 'styled-components'
+import { Link } from 'src/components/Link/Link'
 import { PlotCardTitle } from 'src/components/Common/PlotCardTitle'
 import {
   ClusterDistributionDatum,
   ClusterDistributionPlot,
 } from 'src/components/ClusterDistribution/ClusterDistributionPlot'
+import { gray700 } from '../../theme'
 
 export interface ClusterDistributionPlotCardProps {
-  cluster: string
+  clusterBuildName: string
+  clusterDisplayName: string
   distribution: ClusterDistributionDatum[]
   country_names: string[]
 }
 
+const GreyLink = styled.a`
+  color: ${gray700};
+`
+
 export function ClusterDistributionPlotCard({
-  cluster,
+  clusterBuildName,
+  clusterDisplayName,
   distribution,
   country_names,
 }: ClusterDistributionPlotCardProps) {
+  const url = `/variants/${clusterBuildName}`
+
   return (
     <Card className="m-2">
       <CardHeader className="d-flex flex-sm-column">
         <PlotCardTitle>
-          <span>{cluster}</span>
+          <Link href={url}>
+            <GreyLink>{clusterDisplayName}</GreyLink>
+          </Link>
```

Comment:
`Link` component already contains `a` inside, so this will generate incorrect HTML. Instead, the styling should be applied to the `Link` component directly.

Also, it is preferable to use the function syntax for retrieving theme properties, instead of importing. This way themes can be swapped on runtime.

Example:
https://github.com/hodcroftlab/covariants/blob/cf0283b57f3626adf9513ea506bd93949569a46a/web/src/components/ClusterButtonPanel/ClusterButton.tsx#L41-L71

Reply:
Fixed this in [`f1c8a00` (#92)](https://github.com/hodcroftlab/covariants/pull/92/commits/f1c8a000b7589a485b3e082b36ad67e5d962acab)